### PR TITLE
fix(DTFS2-6655): createGefDeal cypress command - individual deals

### PR DESCRIPTION
**Pull request by vamshidharukef**
## Introduction :pencil2:
For some of the e2e tests after validation is put on users, some tests fail as users with role maker cannot access end point http://localhost:5001/v1/users

## Resolution :heavy_check_mark:
Create cy.createBssApplicationReadyForReview() for creating BSS/EWCS deals through UI
Update commnds.js to use cy.createBssApplicationReadyForReview

### CODE REVIEW

**Review by Zainzzkk — COMMENTED** [review]
Can this be added to the tests where it is required please?

**Review by ttbarnes — CHANGES_REQUESTED** [review]

File: e2e-tests/portal/cypress/support/portal/createGefDealReadyForCheck.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+const facilityCurrency = require('../../../../gef/cypress/e2e/pages/facility-currency');
+const submitToUkef = require('../../../../gef/cypress/e2e/pages/submit-to-ukef');
+const login = require('../../../../gef/cypress/support/commands/portal/login');
+const MOCK_USERS = require('../../../../e2e-fixtures/index');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+/** * Create a BSS/EWCS deal via the UI with a "ready for check" status */
+
+const createGefDealReadyForCheck = () => {
```
[the remark this answers is not in the document]
Comment (Mihail1Ynakiev) [reply e2e-tests/portal/cypress/support/portal/createGefDealReadyForCheck.js:24]: > Same comment as in the BSS/EWCS PR please https://github.com/UK-Export-Finance/dtfs2/pull/3454#discussion_r1752054985

**Comment by ttbarnes** [comment]
TODO - create a new branch and PR with DTFS2-7434 ticket.

**Comment by ttbarnes** [comment]
Closing in favour of https://github.com/UK-Export-Finance/dtfs2/pull/3632

**Pull request closed without merging**